code/modules/mob/hear_say.dm

```diff
@@ -120,14 +120,15 @@
 	else
 		speaker_name = "unknown"
 
-	if(vname)
-		speaker_name = vname
-
 	if(istype(speaker, /mob/living/carbon/human))
 		var/mob/living/carbon/human/H = speaker
 		if(H.voice)
 			speaker_name = H.voice
 
+	if(vname)
+		speaker_name = vname
+
+
```

Comment:
Bit nitpicky, but don't use Nanako-level spacing. One line of whitespace between things at maximum unless you have a really good reason, which I don't see here.